frontend/src/components/profile/EditProfile.vue

```diff
@@ -119,12 +118,24 @@ export default {
             birthMonth: '',
             birthYear: '',
             files: [],
-            oldPassword: '',
-            newPassword: '',
+            user: {
+                id: null,
+                date_of_birth: '',
+                avatar_url: '',
+                email: '',
+                first_name: '',
+                last_name: '',
+                phone_number: '',
+                instagram_url: '',
+                facebook_url: '',
+                twitter_url: '',
+                oldPassword: '',
+                newPassword: '',
+            }
         };
     },
     created() {
-        this.setUser(Object.assign({}, this.authUser));
+        Object.assign(this.user, this.authUser);
```

Comment:
This
`this.user = Object.assign({}, this.authUser);`
this is more better and explicit